### PR TITLE
Update GCE docs

### PR DESCRIPTION
**Pull request by krishnanrs**
Added a networking required section in the GCE page, since they
differ from other CSPs. Also added more descriptions about
optional arguments that can be used with the GCE image encryption
and update process.

### CODE REVIEW

**Comment by robertabbott** [comment]
lgtm

File: gce.md [hunk 1]
```diff
@@ -20,8 +20,42 @@ optional arguments:
...
+Google Cloud Storage
+1. Create an encryptor image locally from the latest Metavisor image
+1. Launch an instance based on the unencrypted image. We call this the
+guest instance.
+1. Snapshot the root volume of the guest instance.
+1. Launch a Bracket Encryptor instance based on the locally created
+encryptor image
+1. Attach the unencrypted guest root volume to the Bracket Encryptor instance
+1. Copy the unencrypted root volume to a new, encrypted volume
+1. Create a new image based on the encryptor image with an appropriate name
```
Comment (bburtin) [inline gce.md]: Not sure if there's a way to clarify "appropriate name."  If not, maybe leave it out?

File: gce.md [hunk 2]
```diff
@@ -164,13 +198,21 @@ optional arguments:
...
+[gcloud](https://cloud.google.com/sdk/gcloud/) and configure it to work 
+with your Google account and GCP project. Make sure that your Google 
+account has `Editor` permissions within the selected Google project.
 
-You'll also need to add a Firewall rule that allows inbound access
+You can use the `--network` option to launch the encryptor instance in
+a specific GCE network. Additionally if you created this network using
+custom subnetworks, then you **must** specify the corresponding subnetwork
+using the `--subnetwork` option. In the absence of the `--network` option,
+the encryptor would be launched in the `default` GCE network.
```
Comment (bburtin) [inline gce.md]: "is launched" reads better than "would be launched"

File: gce.md [hunk 3]
```diff
@@ -164,13 +198,21 @@ optional arguments:
...
+
+You will also need to add a Firewall rule that allows inbound access
```
Comment (bburtin) [inline gce.md]: I don't think that Firewall is a proper noun.  Lower case it?

**Comment by krishnanrs** [comment]
Please take a look again

**Comment by krishnanrs** [comment]
Can someone review again or approve this change?

**Pull request merged**